nncf/quantization/quantize_model.py

```diff
@@ -299,7 +302,7 @@ def compress_weights(
 
         return compress_weights_impl(model, mode, ratio, group_size, ignored_scope)
```

Comment:
Should we add a warning/error that option is not supported in the TORCH backend?

Reply:
thanks, makes sense